src/database/Database.cpp

```diff
@@ -290,6 +323,98 @@ Database::upgradeToCurrentSchema()
     assert(vers == SCHEMA_VERSION);
 }
 
+void
+Database::addTextColumn(std::string const& table, std::string const& column)
+{
+    std::string addColumnStr("ALTER TABLE " + table + " ADD " + column +
+                             " TEXT;");
+    CLOG(INFO, "Database") << "Adding column with string '" << addColumnStr
```

Comment:
Okay.  I did it that way because I occasionally wrote syntax errors and wanted to see exactly what I'd done wrong (the `soci` exceptions generally don't say).  But it makes sense that for a final version, operators wouldn't care.

Reply:
I've updated the log messages in that file to try to constrain them to "what" and not "how".